ui/ui-components/eventHandlers/index.ts

```diff
@@ -0,0 +1,35 @@
+import { ErrorHandler } from "../utils/errorHandler";
+import { SuccessHandler } from "../utils/successHandler";
+import { AppHandler } from "../utils/appHandler";
+import { DestinationHandler } from "../utils/destinationHandler";
+import { GroupHandler } from "../utils/groupHandler";
+import { RecordHandler } from "../utils/recordHandler";
+import { RecordsHandler } from "../utils/recordsHandler";
+import { PropertiesHandler } from "../utils/propertiesHandler";
+import { RunsHandler } from "../utils/runsHandler";
+import { ScheduleHandler } from "../utils/scheduleHandler";
+import { SessionHandler } from "../utils/sessionHandler";
+import { SetupStepHandler } from "../utils/setupStepsHandler";
+import { StatusHandler } from "../utils/statusHandler";
+import { SourceHandler } from "../utils/sourceHandler";
+import { TeamHandler } from "../utils/teamHandler";
+import { TeamMemberHandler } from "../utils/teamMembersHandler";
+import { UploadHandler } from "../utils/uploadHandler";
```

Comment:
Eventually, the event handlers will be moved to this subfolder but while we're referencing these types everywhere this may cause a lot of file changes. So I will keep them in utils for now.